tdrs-backend/wait_for_services.sh

```diff
@@ -5,14 +5,19 @@ wait-for-it \
     --timeout 60 \
     -- echo "ClamAV and Localstack are ready!"
 
-# Wait for Localstack to create the necessary S3 bucket
-echo "Waiting 60 seconds for S3 bucket creation"
-# shellcheck disable=SC2091
-until $(awslocal s3api head-bucket --bucket "$AWS_BUCKET" > /dev/null 2>&1); do
-    ((c++)) && ((c==10)) && echo "S3 bucket not found" && exit 1;
-    sleep 6;
-done
-echo "S3 Bucket is ready after $((c*6)) seconds"
+# Wait for Localstack to create the necessary S3 bucket, if in use
+if [ -z "$USE_LOCALSTACK" ] || [ "$USE_LOCALSTACK" = "yes" ]; then
```

Comment:
Prevents an error if this environment variable is not set, and proceeds with waiting for localstack since that is the default in the context this is run.